src/pages/council.js

```diff
@@ -43,55 +43,56 @@ const CouncilPage = () => {
         <ul>
           <li>Andras Lasso - Queen's University</li>
           <li>Jon Haitz Legarreta Gorroño - Brigham and Women&#39;s Hospital, Harvard Medical School</li>
-          <li>Ipek Oguz - University of Pennsylvania </li>
+          <li>Ipek Oguz - University of Pennsylvania</li>
           <li>Dženan Zukić - Kitware</li>
           <li>Julien Jomier - NVIDIA</li>
           <li>Serena Bonaretti - Balgrist Campus</li>
         </ul>
         <p>Past Council Members include:</p>
         <ul>
-          <li>Paul Yushkevich - UPenn</li>
+          <li>Paul Yushkevich - University of Pennsylvania</li>
           <li>Lauren O'Donnell - Brigham and Women&#39;s Hospital - Harvard Medical School</li>
           <li>Beatriz Paniagua - Kitware</li>
 
-          <li>Elsa Angelini - Columbia</li>
-          <li>Brian Avants - UPenn</li>
-          <li>Stephen Aylward - UNC (now with Kitware)</li>
+          <li>Elsa Angelini - Columbia University</li>
+          <li>Brian Avants - University of Pennsylvania</li>
+          <li>Stephen Aylward - University of North Carolina at Chapel Hill</li>
           <li>Dan Blezek - GE CRD</li>
```

Comment:
Perhaps expand this to `General Electric - Corporate R&D`, also in a couple more instances.

Reply:
Done in 74c16c3.